opentelemetry-stackdriver/src/lib.rs

```diff
@@ -657,3 +652,223 @@ pub enum MonitoredResource {
 
 const TRACE_APPEND: &str = "https://www.googleapis.com/auth/trace.append";
 const LOGGING_WRITE: &str = "https://www.googleapis.com/auth/logging.write";
+const HTTP_PATH_ATTRIBUTE: &str = "http.path";
+use lazy_static::lazy_static;
+
+lazy_static! {
```

Comment:
Unless you have actual benchmark data showing that this helps, I'd prefer to just have these as `const &str`s, with the `From` impl calling `to_owned()` on those.

Reply:
Give me a couple of hours and I'll push a change with `const &str`. I preferred it that way but wasn't sure as saw a lot of `const` in other exporters for fields like this. Also, the change means I can use a `match` rather than the horrible if statement I've got.

Reply:
Ha, yeah! I just reverted my change and remembered why I wasn't using a matcher 😅